### ISSUE TITLE
Make use of Conda optional

### ISSUE THREAD

**Issue by Peque** [tox-dev/tox-conda#14]
It would be great if, by default, `tox-conda` would not alter the default `tox` behavior. That way you could have it installed and not force its use in potentially many different projects.

I propose this as I have `tox` installed user-wide, not in a virtual environment, so I would like to do the same with `tox-conda` without breaking many projects which do not (and should not) use Conda at all.

Example:

```yaml
# This would "not activate" `tox-conda` (standard `tox` behavior)
[testenv]
deps = whatever
```

```yaml
# This would "activate" `tox-conda` (specific `conda` reference detected)
[testenv]
conda_deps = whatever
```

```yaml
# This would "activate" `tox-conda` too (explicit setting)
[testenv]
use_conda = True
deps = whatever
```

**Comment by Peque** [comment]
This could maybe one day help integrating Conda support in Tox, as it would be backwards compatible, not breaking people's workflows?

**Comment by AnneTheAgile** [comment]
I believe this issue as well should be closed as #21 was merged.
